.babelrc

```diff
@@ -1,13 +1,11 @@
 {
-	"presets": ["env"],
+	"presets": [
+		"@babel/preset-env"
+	],
 	"plugins": [
 		"transform-object-assign",
 		"transform-es3-property-literals",
-		"transform-es3-member-expression-literals"
-	],
-	"env": {
-    "test": {
-      "plugins": [ "istanbul" ]
-    }
-  }
+		"transform-es3-member-expression-literals",
+		"@babel/plugin-transform-modules-commonjs"
```

Comment:
Why do we need `@babel/plugin-transform-modules-commonjs` ?

Reply:
@jaiminpanchal27 We updated the version of `babel-preset-env`, the plugin to transform es6 modules to commonjs modules no longer comes bundled with the new version. We need to install it separately. Since, some of our files are using `import/export` syntax need this plugin or else, the build fails with the error, `exports is not defined.`